### ISSUE TITLE
Write page "How to set up LDAP authentication for The Lounge"

### ISSUE THREAD

**Issue by xPaw** [thelounge/thelounge.github.io#145]

**Comment by astorije** [comment]
Hey @gramakri, do you think you could help us with that? You don't have to write the full thing, but if you had bullet points or anything to help me write it, that'd be super helpful. Happy to write it with you.

**Comment by gramakri** [comment]
Yup, I can write a doc. I should submit a PR to this directory https://github.com/thelounge/thelounge.github.io/tree/master/_guides correct?

**Comment by xPaw** [comment]
Yes.

**Comment by astorije** [comment]
@gramakri, correct, I've set up a placeholder at https://github.com/thelounge/thelounge.github.io/blob/astorije/ldap/_guides/ldap.md (result at https://astorije-ldap--thelounge.netlify.com/docs/guides/ldap), feel free to open a PR against my branch `astorije/ldap` (then I can squash my commit into your stuff).

In case you can't run Jekyll on your end, simply open a PR and there will be a deploy preview (when Netlify fixes them :D).

**Comment by gramakri** [comment]
Sorry for the delay, I made a PR now.
